labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -32,8 +32,19 @@ object CollectionExercise01 {
    * Case 3: so it is okay if you want to just give up
    *
    */
-  def googleCodeJamGooglerese(lines: String*): Seq[String] = {
-    error("fix me")
+  
```

Comment:
Woah, this is a pretty cool solution. It looks a bit simpler than the example solution if I remember right. Nice! Like the use of dictionaries.